libraries/AP_DDS/README.md

```diff
@@ -253,7 +253,28 @@ requester: making request: ardupilot_msgs.srv.ModeSwitch_Request(mode=4)
 response:
 ardupilot_msgs.srv.ModeSwitch_Response(status=True, curr_mode=4)
 ```
- 
+## Commanding using ROS 2 Topics
+The following topic can be used to control the vehicle.
```

Comment:
Add to `published topics` section on line 185 too.

Reply:
that is already mentioned in the Subscribed topics